### ISSUE TITLE
Hi, I have a question about actor loss.

### ISSUE THREAD

**Issue by wenghongda** [qingshi9974/PPO-pytorch-Mujoco#6]
actor_loss = -torch.min(surrogate_loss,clipped_loss).mean()
Why the there is a negative sign before the torch.min?
